middleware/controllers.js

```diff
@@ -17,6 +18,17 @@ module.exports = function(app, fundation) {
 
   debug("Setting up Controllers");
 
+  /**
+   * registerHook
+   *
+   * @param  {string} file name of file to register
+   */
+  var registerHook = function(file) {
+    if ( fs.existsSync(path.resolve('controllers/' + file + '.js')) ) {
+      await (require(path.resolve('controllers/' + file + '.js'))(app, fundation));
+    }
+  }
+
```

Comment:
was wondering if I should move the function out and include `app`, and `fundation`
for example
``` js
function registerHook(file, app, fundation) {
  ...
}
```